src/stronghold/secret.rs

```diff
@@ -36,35 +38,35 @@ impl SecretManage for StrongholdAdapter {
         _metadata: GenerateAddressMetadata,
     ) -> Result<Vec<Address>> {
         // Stronghold arguments.
-        let seed_location = SLIP10DeriveInput::Seed(Location::Generic {
+        let seed_location = Slip10DeriveInput::Seed(Location::Generic {
```

Comment:
`Location` offers some convenience constructors like `Location::generic<V: Into<Vec<u8>>, R: Into<Vec<u8>>>(vault_path: V, record_path: R) -> Self` So you don't need to explicitly convert to `Vec<u8>`